theano/tensor/nnet/abstract_conv.py

```diff
@@ -90,26 +95,31 @@ def get_conv_shape_1axis(image_shape, kernel_shape,
         the padding on the considered axis.
     subsample: int. It must correspond to the subsampling on the
         considered axis.
+    dilation: int. It must correspond to the dilation on the
+        considered axis.
 
     Returns
     -------
     out_shp: int corresponding to the output image shape on the
         considered axis. None if undefined.
 
     """
-    if None in [image_shape, kernel_shape, border_mode, subsample]:
+    if None in [image_shape, kernel_shape, dilation,
+                border_mode, subsample]:
```

Comment:
I'd have expected the same order as in the argument list, but that's not critical.